app/components/support_interface/reference_with_feedback_component.html.erb

```diff
@@ -1,5 +1,5 @@
 <div data-qa="reference">
-  <%= render SummaryCardComponent.new(rows: rows, editable: true) do %>
+  <%= render SummaryCardComponent.new(rows: rows, editable: true, warning_text: warning_text) do %>
```

Comment:
```suggestion
  <%= render SummaryCardComponent.new(rows:, editable: true, warning_text:) do %>
```
Not a blocker, but you shouldn't have to repeat yourself, `rows: rows`, `warning_text: warning_text`

Reply:
Thanks. I've removed those here: https://github.com/DFE-Digital/apply-for-teacher-training/pull/10137/commits/492ad33b92b03ea388fa00c7204ba4ca10b80b8b